projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -11,10 +11,20 @@
 use Automattic\Jetpack\Status;
 use Jetpack;
 use Jetpack_Gutenberg;
+use Jetpack_Subscriptions_Widget;
 
 const FEATURE_NAME = 'subscriptions';
 const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
 
+/**
+ * These block defaults should match ./constants.js
+ */
+const DEFAULT_BORDER_RADIUS_VALUE = 0;
+const DEFAULT_BORDER_WEIGHT_VALUE = 1;
```

Comment:
Is this actually "weight"? Should it be "width"?

Reply:
It doesn't really matter since it's just a PHP constant, but the css prop is indeed `border-width`.  
As a good part of this code was migrated from the shortcode implementation, I opted not to make too many changes for now and left this one as is.